src/library/baseplaylistfeature.cpp

```diff
@@ -82,6 +83,10 @@ BasePlaylistFeature::BasePlaylistFeature(QObject* parent,
 
     connect(&m_playlistDao, SIGNAL(lockChanged(int)),
             this, SLOT(slotPlaylistTableChanged(int)));
+
+    Library* pLibrary = static_cast<Library*>(parent);
```

Comment:
I think this is redundant with library.cpp:229 -- all LibraryFeatures should already get this signal. Did you find that it wasn't being delivered?